src/main/kotlin/traffic_simulation/Vehicle.kt

```diff
@@ -1,31 +1,10 @@
 package traffic_simulation
 
-class Vehicle(val id : Int, val wannaDrive : Boolean, var delayed : Boolean = false, var hasDriven : Boolean = false){
-    val allVehicles : MutableList<Vehicle> = mutableListOf()
+class Vehicle(val id : Int, val wannaDrive : Boolean){
+    var delayed : Boolean = false //no vehicle is delayed by default
```

Comment:
Ich fände es schöner wenn `delayed` ein Attribut von der Klasse `Vehicle` wäre. Den Default `delayed : Boolean = false` konnte man ja wie vorher auch in den Konstruktor rein machen.